### PR TITLE
Qfix/floating wireguard example

### PR DESCRIPTION
**Pull request by Mixaster995**
Signed-off-by: Mikhail Avramenko <avramenkomihail15@gmail.com>

### CODE REVIEW

File: examples/interdomain/usecases/FloatingKernel2Wireguard2Kernel/README.md [hunk 1]
```diff
@@ -58,12 +58,17 @@ metadata:
   name: nse-kernel
 spec:
   template:
+    metadata:
+      annotations:
+        registration-name: icmp-server@my.cluster3
```
Comment (edwarnicke) [inline examples/interdomain/usecases/FloatingKernel2Wireguard2Kernel/README.md:63]: Why are we adding an annotation for registration-name here?

Comment (denis-tingaikin) [reply examples/interdomain/usecases/FloatingKernel2Wireguard2Kernel/README.md:63]: I guess there is a problem with kustomize merge. OK, we'll look at a better solution for it.

Comment (edwarnicke) [reply examples/interdomain/usecases/FloatingKernel2Wireguard2Kernel/README.md:63]: What is the problem with kustomize merge?

Comment (denis-tingaikin) [reply examples/interdomain/usecases/FloatingKernel2Wireguard2Kernel/README.md:63]: @edwarnicke See at https://github.com/networkservicemesh/deployments-k8s/pull/4057

**Pull request closed without merging**